app.js

```diff
@@ -0,0 +1,84 @@
+function createBook(title, author) {
```

Comment:
- [ ] [OPTIONAL] Kindly use arrow functions for your functions, this will help you in the future because it's a good ES6 feature, You can learn more about arrow functions [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)